UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -50,6 +54,8 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double FootPoundsInOneJoule => 0.737562149;
 
+        protected override double GigabritishThermalUnitsInOneJoule => 9.4781339449889105832843629746176e-13;
```

Comment:
Actually, this one seems to still use the old constant?

Reply:
Oops... my bad.